test/info.py

```diff
@@ -1,6 +1,6 @@
 #
 # DAPLink Interface Firmware
-# Copyright (c) 2009-2016, ARM Limited, All Rights Reserved
+# Copyright (c) 2009-2019, ARM Limited, All Rights Reserved
```

Comment:
Please revert changing permissions to this file (100644 → 100755)

Reply:
Few files in this PR

Reply:
changed